src/io/flutter/view/InspectorPanel.java

```diff
@@ -645,30 +646,59 @@ protected void customizeCellRenderer(JTable table, @Nullable Object value, boole
         append(node.getName(), textAttributes);
         append(" ", textAttributes);
       }
-      // TODO(jacobr): also provide custom UI display for padding, transform,
-      // and alignment properties.
+
+      boolean appendDescription = true;
+
+      // TODO(jacobr): also provide custom UI display for padding, transform, and alignment properties.
       final CompletableFuture<Map<String, InstanceRef>> propertiesFuture = node.getValueProperties();
       if (propertiesFuture.isDone() && !propertiesFuture.isCompletedExceptionally()) {
         final Map<String, InstanceRef> properties = propertiesFuture.getNow(null);
         switch (node.getPropertyType()) {
           case "Color": {
-            final int alpha = getIntProperty(properties, "alpha");;
-            final int red = getIntProperty(properties, "red");;
-            final int green = getIntProperty(properties, "green");;
-            final int blue = getIntProperty(properties, "blue");;
+            final int alpha = getIntProperty(properties, "alpha");
+            final int red = getIntProperty(properties, "red");
+            final int green = getIntProperty(properties, "green");
+            final int blue = getIntProperty(properties, "blue");
 
+            //noinspection UseJBColor
             final Color color = new Color(red, green, blue, alpha);
             this.setIcon(colorIconMaker.getCustomIcon(color));
             if (alpha == 255) {
               append(String.format("#%02x%02x%02x", red, green, blue), textAttributes);
-            } else {
+            }
+            else {
               append(String.format("#%02x%02x%02x%02x", alpha, red, green, blue), textAttributes);
             }
+
+            appendDescription = false;
+
+            break;
+          }
+
+          case "IconData": {
+            final String valueToString = node.getStringMember("valueToString");
```

Comment:
Instead of using valueToString
a more robust option is to use
getIntProperty(properties, "codePoint") 
as in the "Color" case above.
You'll need to repeat yourself in DiagnosticsNode.java indicating that you want the "codePoint" Dart property but you should just be able to search for "Color" to see what you need to copy.
Should be more robust than the string parsing. Feel free to also checkin like this and add a follow up CL that using getIntProperty or I can add that CL.

Reply:
I don't think I saw that property (`codePoint`) when debugging, but I'll double check.